docs/source/pages/developers_guide/index.rst

```diff
@@ -0,0 +1,18 @@
+.. Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+=========================
+Macaron Developer's Guide
+=========================
+
+To get started with contributing to Macaron, see our `CONTRIBUTING <https://github.com/oracle/macaron/blob/main/CONTRIBUTING.md>`_ page.
+
+To follow our code style, see our :doc:`Macaron Style Guide </pages/developers_guide/style_guide>` page.
```

Comment:
```suggestion
To follow our code style, see the :doc:`Macaron Style Guide </pages/developers_guide/style_guide>` page.
```

Reply:
See 90decf0.

Reply:
Fixed in 90decf08e1ed1f1bb48033b713d067ac96910d31.